website/docs/r/is_lb_pool_member.html.markdown

```diff
@@ -63,7 +63,7 @@ Review the argument references that you can specify for your resource.
 - `port`- (Required, Integer) The port number of the application running in the server member.
 - `target_address` - (Required, String) The IP address of the pool member.
 - `target_id` - (Required, String) The unique identifier for the virtual server instance pool member. Required for network load balancer.
-- `weight` - (Optional, Integer) Weight of the server member. This option takes effect only when the load-balancing algorithm of its belonging pool is `weighted_round_robin`, Minimum allowed weight is `0` and Maximum allowed weight is `100`.
+- `weight` - (Optional, Integer) Weight of the server member. This option takes effect only when the load-balancing algorithm of its belonging pool is `weighted_round_robin`, Minimum allowed weight is `0` and Maximum allowed weight is `100`. When weight is not provided a default of 40 is returned.
```

Comment:
in test case you said default is 50 but in docs its 40 can you confirm

Reply:
modified with right info.